website/source/api/catalog.html.md

```diff
@@ -106,8 +113,15 @@ and vice versa. A catalog entry can have either, neither, or both.
     "Name": "Redis health check",
     "Notes": "Script based health check",
     "Status": "passing",
-    "ServiceID": "redis1"
-  }
+    "ServiceID": "redis1",
+    "Definition": {
+      "TCP": "localhost:8888",
+      "Interval": "5s",
+      "Timeout": "1s",
+      "DeregisterCriticalServiceAfter": "30s"
```

Comment:
We should add a TODO to the ESM to support `DeregisterCriticalServiceAfter` thing (not important right now but nice to add for later).